flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/datagenerator/common/ColsWithLabelGenerator.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Let's add `[FLINK-27096]` to the title and commit message of this PR. It has been the parent ticket for all PRs related to algorithm benchmark and performance improvement before 2.1 release.